core/src/main/scala/kafka/server/KafkaRaftServer.scala

```diff
@@ -185,7 +185,7 @@ object KafkaRaftServer {
     val bootstrapMetadata = if (config.originals.containsKey(KafkaConfig.InterBrokerProtocolVersionProp)) {
       BootstrapMetadata.load(Paths.get(config.metadataLogDir), config.interBrokerProtocolVersion)
     } else {
-      BootstrapMetadata.load(Paths.get(config.metadataLogDir), MetadataVersion.IBP_3_0_IV0)
+      BootstrapMetadata.load(Paths.get(config.metadataLogDir), MetadataVersion.MINIMUM_KRAFT_VERSION)
```

Comment:
nit: the use of "preview" in some of the comments and code seems unnecessary and likely to cause confusion. Could we just say older versions? Instead of `fallbackPreviewVersion`, maybe just `fallbackVersion`?

Reply:
Agreed on the "preview" terminology. I've fixed all the ones I could find